plugins/generic/customBlockManager/SettingsForm.inc.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * @file SettingsForm.inc.php
+ *
+ * Copyright (c) 2003-2015 John Willinsky. For full terms see the file docs/COPYING.
+ *
+ * @class SettingsForm
+ *
+ * @brief Form for site admins to add or delete sidebar blocks
```

Comment:
missing package info, @file should have the complete path, different copyright text